prowler/providers/aws/services/cloudformation/cloudformation_stack_outputs_find_secrets/cloudformation_stack_outputs_find_secrets.py

```diff
@@ -27,7 +27,11 @@ def execute(self):
                     data += f"{output}\n"
 
                 detect_secrets_output = detect_secrets_scan(
-                    data=data, excluded_secrets=secrets_ignore_patterns
+                    data=data,
+                    excluded_secrets=secrets_ignore_patterns,
+                    detect_secrets_plugins=cloudformation_client.audit_config.get(
```

Comment:
I think you should get the default list of plugins instead of `None` or `{}`. Please review it in all checks.

Reply:
I agree with you, changes on the next commit.